packages/react-model-ui/src/views/common/Icons.tsx

```diff
@@ -3,14 +3,16 @@
  ********************************************************************************/
 /* eslint-disable max-len */
 
-import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
-import React = require('react');
+import * as React from 'react';
 
-export const KeyIcon: React.FC<SvgIconProps> = props => (
+interface IconProps {
+   className?: string;
+   style?: React.CSSProperties;
+}
+
+export const KeyIcon: React.FC<IconProps> = props => (
```

Comment:
This seems to be unused? Either we should remove the file or we render the icon as part of the column header in the entity attributes.